extension/sidebar-pane.js

```diff
@@ -62,24 +62,71 @@ async function _updateSubtree(selectedNode) {
 
     const { attributes, nodeName } = node;
     issues.push(
-      ..._webcompat.getHTMLElementIssues(nodeName, attributes, _targetBrowsers));
+      ...
+       _webcompat.getHTMLElementIssues(nodeName, attributes, _targetBrowsers)
+                 .map(issue => {
+                   issue.node = node;
+                   return issue;
+                 })
+    )
   }
 
   progressEl.textContent = "Getting all descendants of the selected node";
   const declarationBlocks = await browser.experiments.inspectedNode.getStylesInSubtree();
 
   progressEl.textContent = "Getting web compatibility issues for CSS styles";
-  for (const { declarations } of declarationBlocks) {
+  for (const { node, declarations } of declarationBlocks) {
     issues.push(
-      ..._webcompat.getCSSDeclarationBlockIssues(declarations, _targetBrowsers));
+      ...
+       _webcompat.getCSSDeclarationBlockIssues(declarations, _targetBrowsers)
+                 .map(issue => {
+                   issue.node = node;
+                   return issue;
+                 })
+    )
   }
 
+  progressEl.textContent = "Grouping all issues";
+  const issueGroups = _grouping(issues);
+
   progressEl.textContent = "Rendering all issues";
-  _render(issues, issueListEl);
+  _render(issueGroups, issueListEl);
 
   subtreeEl.classList.remove("processing");
 }
 
+function _grouping(issues) {
```

Comment:
This method would be more clearer named as `groupIssues()`. 
Please add a JSDoc comment to the method to briefly explain what goes in and what is returned.